test/unit/BondingManager.js

```diff
@@ -639,6 +639,93 @@ describe("BondingManager", () => {
             beforeEach(async () => {
                 await bondingManager.connect(delegator).bond(2000, transcoder0.address)
             })
+
+            describe("caller is transferring bond to another address", () => {
```

Comment:
I think it would be clearer if the test cases specific to `transferBond()` are grouped under their own describe block. This would make it easier to review testing of functionality that is specific to `transferBond()` (i.e. the new cases added thus far here). WDYT?